kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -13,9 +13,32 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 	
-	
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask.png"></p>
```

Comment:
please show only the properties panel to be in sync with other screenshots

Reply:
Hi @radtriste can you explain, sorry I did not understand

Reply:
same as the other one. you can leave it like that.
I was just saying that all screenshots are on the properties panel but some have the diagram also with.
It can be quite confusing but is a minor detail. As long as it is readable ;)

Reply:
fixed @radtriste